pkg/pointerdb/pdbclient/client.go

```diff
@@ -26,7 +28,7 @@ var (
 // PointerDB creates a grpcClient
 type PointerDB struct {
 	client        pb.PointerDBClient
-	authorization *pb.SignedMessage
+	authorization unsafe.Pointer
```

Comment:
Adding a comment of what type is actually stored helps to understand code better. e.g. `authorization unsafe.Pointer // *pb.SignedMessage`

Reply:
Yes, makes sense.